docs/tutorials/send_receive_ingress_pem.md

```diff
@@ -0,0 +1,364 @@
+---
+title: "Exchanging messages over an ssl ingress with cert-manager's generated certs"
+description: "Steps to get a producer and a consummer exchanging messages over a deployed broker on kubernetes using an ingress and certificates genererated via cert-manager"
+draft: false
+images: []
+menu:
+  docs:
+    parent: "tutorials"
+weight: 110
+toc: true
+---
+
+### Prerequisite
+
+Before you start, you need to have access to a running Kubernetes cluster
+environment. A [Minikube](https://minikube.sigs.k8s.io/docs/start/) instance
+running on your laptop will do fine.
+
+#### Start minikube with a parametrized dns domain name
+
+```console
+$ minikube start --dns-domain='demo.artemiscloud.io'
+😄  minikube v1.32.0 on Fedora 39
+🎉  minikube 1.33.1 is available! Download it: https://github.com/kubernetes/minikube/releases/tag/v1.33.1
+💡  To disable this notice, run: 'minikube config set WantUpdateNotification false'
+
+✨  Automatically selected the kvm2 driver. Other choices: qemu2, ssh
+👍  Starting control plane node minikube in cluster minikube
+🔥  Creating kvm2 VM (CPUs=2, Memory=6000MB, Disk=20000MB) ...
+🐳  Preparing Kubernetes v1.28.3 on Docker 24.0.7 ...
+    ▪ Generating certificates and keys ...
+    ▪ Booting up control plane ...
+    ▪ Configuring RBAC rules ...
+🔗  Configuring bridge CNI (Container Networking Interface) ...
+    ▪ Using image gcr.io/k8s-minikube/storage-provisioner:v5
+🔎  Verifying Kubernetes components...
+🌟  Enabled addons: storage-provisioner, default-storageclass
+🏄  Done! kubectl is now configured to use "minikube" cluster and "default" namespace by default
+```
+
+#### Enable nginx and ssl passthrough for minikube
+
+```console
+$ minikube addons enable ingress
+$ minikube kubectl -- patch deployment -n ingress-nginx ingress-nginx-controller --type='json' -p='[{"op": "add", "path": "/spec/template/spec/containers/0/args/-", "value":"--enable-ssl-passthrough"}]'
+```
+
```

Comment:
add some reference to the lovely new hostname_resoultion doc here or close to here.